firewood/src/db.rs

```diff
@@ -372,6 +372,15 @@ impl<S: ShaleStore<Node> + Send + Sync> DbRev<S> {
         self.merkle.prove::<K>(key, self.header.kv_root)
     }
 
+    pub fn verify_proof<K: AsRef<[u8]>, N: AsRef<[u8]> + Send>(
```

Comment:
The `verify` method on proof was removed so we need this in order for someone with a `DbRev` to verify a proof